### PR TITLE
Document Collapser/CollapserGroup + other changes

### PR DESCRIPTION
**Pull request by jerelmiller**
[Rendered](https://github.com/newrelic/gatsby-theme-newrelic/blob/jerel/document-new-stuff/packages/gatsby-theme-newrelic/README.md)

### CODE REVIEW

File: packages/gatsby-theme-newrelic/README.md [hunk 1]
```diff
@@ -656,6 +658,80 @@ const Documentation = () => (
...
+
+```js
+import { Collapser } from '@newrelic/gatsby-theme-newrelic'`
+```
+
+**Props**
+
+| Prop          | Type                    | Required | Default | Description                                                                                                                  |
+| ------------- | ----------------------- | -------- | ------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `children`    | enum                    | yes      |         | The content that will be hidden or revealed when the user interacts with the `Collapser`.                                    |
```
Comment (zstix) [inline packages/gatsby-theme-newrelic/README.md:675]: Shouldn't `children` be of the `node` type? If not, we should update the propTypes in the component.

**Review by zstix — COMMENTED** [review]
Looks great, just one minor question!

File: packages/gatsby-theme-newrelic/README.md (hunk 1 above)
Comment (jerelmiller) [reply packages/gatsby-theme-newrelic/README.md:675]: Oops, good catch. I copy/pasted this from another component and forgot to update this one.

Comment (jerelmiller) [reply packages/gatsby-theme-newrelic/README.md]: Fixed in 94a8519

**Review by zstix — APPROVED** [review]

**Pull request merged**